fabric-object-builder-api-v1/src/testmod/resources/fabric.mod.json

```diff
@@ -26,6 +26,9 @@
       "net.fabricmc.fabric.test.object.builder.CriterionRegistryTest::init",
       "net.fabricmc.fabric.test.object.builder.VillagerTypeTest1",
       "net.fabricmc.fabric.test.object.builder.VillagerTypeTest2"
+    ],
+    "fabric-gametest": [
+      "net.fabricmc.fabric.test.object.builder.BlockEntityTypeBuilderTest"
```

Comment:
No idea if putting the game tests in the same file as the mod initializer is good practice (doesn't seem so).